starknet_py/constants.py

```diff
@@ -16,4 +16,14 @@
 RPC_INVALID_MESSAGE_SELECTOR_ERROR = 21
 RPC_CLASS_HASH_NOT_FOUND_ERROR = 28
 
+DEFAULT_ENTRY_POINT_NAME = "__default__"
+DEFAULT_L1_ENTRY_POINT_NAME = "__l1_default__"
+DEFAULT_ENTRY_POINT_SELECTOR = 0
+MASK_250 = 2**250 - 1
+
+# MAX_STORAGE_ITEM_SIZE and ADDR_BOUND must be consistent with the corresponding constant in
+# starkware/starknet/common/storage/storage.cairo.
+MAX_STORAGE_ITEM_SIZE = 256
```

Comment:
How likely is that this value will get increased in the future causing bugs in starknet.py? Is there any way to test it except for manual verification (probably not)?

Reply:
It might be changed in long future, but it will be a breaking change in starknet.